### PR TITLE
chore: Refactor Breadcrumb to section

### PR DESCRIPTION
**Pull request by igorbrasileiro**
## What's the purpose of this pull request?

<em>Considering the context, what is the problem we'll solve? Where in VTEX's big picture our issue fits in? Write a tweet about the context and the problem itself.</em>

## How does it work?

<em>Tell us the role of the new feature, or component, in its context.</em>

## How to test it?

<em>Describe the steps with bullet points. Is there any external reference, link, or example?</em>

## References

<em>Spread the knowledge: is this any content you used to create this PR that is worth sharing?</em>

<em>Extra tip: add references to related issues or mention people important to this PR may be good for the documentation and reviewing process</em>

## Checklist

<em>You may erase this after checking them all ;)</em>

- [] CHANGELOG entry added

### CODE REVIEW

File: src/pages/{StoreCollection.slug}.tsx [hunk 1]
```diff
@@ -73,10 +73,12 @@ function Page(props: Props) {
       */}
 
       <div className="product-listing__breadcrumb / grid-content">
```
Comment (hellofanny) [inline src/pages/{StoreCollection.slug}.tsx:75]: I think we can remove this `div` here and add those classes to the `section` below.

File: src/pages/{StoreCollection.slug}.tsx [hunk 2]
```diff
@@ -73,10 +73,12 @@ function Page(props: Props) {
...
-        <BreadcrumbWrapper
-          breadcrumbList={collection?.breadcrumbList.itemListElement}
-          name={title}
-        />
+        <section>
```
Comment (hellofanny) [inline src/pages/{StoreCollection.slug}.tsx:76]:
```suggestion
        <section className="product-listing__breadcrumb / grid-content">
```

File: src/pages/{StoreCollection.slug}.tsx (hunk 1 above)
Comment (igorbrasileiro) [reply src/pages/{StoreCollection.slug}.tsx:75]: Thank you for the suggestion, i will apply

File: src/components/sections/Breadcrumb/Breadcrumb.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import type { BreadcrumbProps } from 'src/components/ui/Breadcrumb'
+import UIBreadcrumb from 'src/components/ui/Breadcrumb'
+
+interface BreadcrumbWrapperProps
+  extends Partial<Pick<BreadcrumbProps, 'breadcrumbList'>> {
+  name: string
+}
+
+function Breadcrumb({ breadcrumbList, name }: BreadcrumbWrapperProps) {
+  const fallback = [{ item: '/', name, position: 1 }]
```
Comment (tlgimenes) [inline src/components/sections/Breadcrumb/Breadcrumb.tsx:11]: please move this to the global scope so we don't re-create this object at every render

**Review by tlgimenes — APPROVED** [review]
after requested changes, LGTM

File: src/components/sections/Breadcrumb/Breadcrumb.tsx (hunk 3 above)
Comment (igorbrasileiro) [reply src/components/sections/Breadcrumb/Breadcrumb.tsx:11]: This array needs to be recreated due to the property name, I thought the same, but the name comes from props.

**Review by hellofanny — APPROVED** [review]

**Pull request merged**